lvm.go

```diff
@@ -130,8 +135,11 @@ type LV struct {
 type LVType int
 
 const (
+	// LVTypeUnknown - unknown type
```

Comment:
I'm going to change this back so that is backwards compat. for now at least.
leaving THICK = 0